lib/gems/pending/appliance_console/cli.rb

```diff
@@ -322,6 +329,25 @@ def extauth_opts
       extauthopts.update_configuration(extauthopts_hash)
     end
 
+    def set_server_state
+      service = LinuxAdmin::Service.new("evmserverd")
+      service_running = service.running?
+      case options[:server]
+      when :status
+        service_running ? "running" : "not running"
+        say "EvmServed is " + service.running?
```

Comment:
The service name is evmserverd - I think this is misspelled and has some odd capitalization.